language/German/strings.po

```diff
@@ -1497,16 +1525,16 @@ msgstr "Klone einschließen"
 
 #: src/frontend/mame/ui/utils.cpp:1131
 msgid "Category File"
-msgstr ""
+msgstr "Kategorie Datei"
 
 #: src/frontend/mame/ui/utils.cpp:2030 src/frontend/mame/ui/inifile.cpp:301
 msgctxt "swlist-info"
 msgid "Software list/item"
-msgstr ""
+msgstr "Software-Liste/Eintrag"
 
 #: src/frontend/mame/ui/datmenu.cpp:88 src/frontend/mame/ui/selmenu.cpp:2685
 msgid "Software List Info"
-msgstr ""
+msgstr "Software-Listen-Information"
```

Comment:
"Softwarelisten-Information"? "Softwarelisten-Info" (to keep the "Info" from the English text) Or "Softwarelisteninfo" ;-) or "Info zur Softwareliste"?

Reply:
Changed